protocol.html

```diff
@@ -406,15 +406,15 @@ <h2 property="schema:name">Hypertext Transfer Protocol</h2>
               <section id="http-server" inlist="" rel="schema:hasPart" resource="#http-server">
                 <h3 property="schema:name">HTTP Server</h3>
                 <div datatype="rdf:HTML" property="schema:description">
-                  <p>A <a data-link-type="dfn" href="#data-pod" id="ref-for-data-pod①">data pod</a> MUST be an HTTP/1.1 server [<cite><a class="bibref" href="#bib-rfc7230">RFC7230</a></cite>][<cite><a class="bibref" href="#bib-rfc7231">RFC7231</a></cite>]. It SHOULD additionally be an HTTP/2 server [<cite><a class="bibref" href="#bib-rfc7540">RFC7540</a></cite>] to improve performance, especially in cases where individual clients are expected to send high numbers of successive requests.</p>
+                  <p>A <span id="sr-http-http-11-server"><a data-link-type="dfn" href="#data-pod" id="ref-for-data-pod①">data pod</a> MUST be an HTTP/1.1 server [<cite><a class="bibref" href="#bib-rfc7230">RFC7230</a></cite>][<cite><a class="bibref" href="#bib-rfc7231">RFC7231</a></cite>]</span>. <span id="sr-http-http-2-server">It SHOULD additionally be an HTTP/2 server [<cite><a class="bibref" href="#bib-rfc7540">RFC7540</a></cite>]</span> to improve performance, especially in cases where individual clients are expected to send high numbers of successive requests.</p>
```

Comment:
Unrelated to this PR, but what is "①" character doing in the `id="ref-for-data-pod①"` element? Is that intentional?

Reply:
I don't know, @edwardsph and I noted it too... We need to bring it to @csarven 's attention when gets back.

Reply:
"①" is garbage from bikeshed. Didn't get around to clean that up - along with adding ids all over the document.